### PR TITLE
Update CHANGELOG.md for Rust 1.43 and 1.44

### PR DESCRIPTION
**Pull request by phansch**
[Rendered](https://github.com/phansch/rust-clippy/blob/update_changelog/CHANGELOG.md)

changelog: none

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,15 +4,98 @@ All notable changes to this project will be documented in this file.
...
+
+### Others
 
 ## Rust 1.43
 
-Current beta, release 2020-04-23
+Current stable, release 2020-04-23
 
-[4ee1206...329923e](https://github.com/rust-lang/rust-clippy/compare/4ee1206...329923e)
+[4ee1206...204bb9b](https://github.com/rust-lang/rust-clippy/compare/4ee1206...204bb9b)
```
Comment (phansch) [inline CHANGELOG.md:98]: The diff between those two is just a removal of the git2 dependency, so not worth a mention in the changelog: https://github.com/rust-lang/rust-clippy/compare/329923e...204bb9b

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,15 +4,98 @@ All notable changes to this project will be documented in this file.
...
+* Fix bit mask example in [`verbose_bit_mask`] documentation [#5454](https://github.com/rust-lang/rust-clippy/pull/5454)
+* [`wildcard_imports`] documentation now mentions that `use ...::prelude::*` is
+  not linted [#5312](https://github.com/rust-lang/rust-clippy/pull/5312)
+
+### Others
 
 ## Rust 1.43
 
-Current beta, release 2020-04-23
+Current stable, release 2020-04-23
```
Comment (flip1995) [inline CHANGELOG.md:96]:
```suggestion
Current stable, released 2020-04-23
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,15 +4,98 @@ All notable changes to this project will be documented in this file.
...
+### Documentation
+
+* Fix documentation generation for configurable lints [#5353](https://github.com/rust-lang/rust-clippy/pull/5353)
+* Update documentation for [`new_ret_no_self`] [#5448](https://github.com/rust-lang/rust-clippy/pull/5448)
+* The documentation for [`option_option`] now suggest using a tri-state enum [#5403](https://github.com/rust-lang/rust-clippy/pull/5403)
+* Fix bit mask example in [`verbose_bit_mask`] documentation [#5454](https://github.com/rust-lang/rust-clippy/pull/5454)
+* [`wildcard_imports`] documentation now mentions that `use ...::prelude::*` is
+  not linted [#5312](https://github.com/rust-lang/rust-clippy/pull/5312)
+
+### Others
```
Comment (flip1995) [inline CHANGELOG.md:92]:
```suggestion
```

**Comment by phansch** [comment]
Ohh, I should also mention that I didn't include the `cargo clippy --fix` changes, because they will still remain nightly only for a while. Let me know if I should mention that here as well =)

**Comment by flip1995** [comment]
I think we should include a line "Nightly only: `cargo clippy --fix -Z unstable-options` now fixes Clippy findings" (or something similar).  I'd like people to discover the `--fix` arg more easily, so we can get feedback and fix potential bugs before stabilizing it. I think putting it in the changelog will help.

Advertising it in the Inside Rust blog sounds also good to me. @yaahc do you want to write the blog post, since you pushed this all the way through? :)

**Comment by phansch** [comment]
Added `--fix` to the top of `Enhancements` :+1:

**Comment by yaahc** [comment]
Sure, I would be happy to write the blog post

**Pull request merged**